sklearn/df_clsf.py

```diff
@@ -3,7 +3,8 @@
 # SPDX-License-Identifier: MIT
 
 import argparse
-from bench import parse_args, time_mean_min, print_header, print_row, size_str
+from bench import parse_args, time_mean_min, print_header, print_row,\
```

Comment:
For these multi-line imports, I would like for the project to follow numpy's practice of using parentheses and indenting the inside. See https://github.com/numpy/numpy/blob/master/numpy/linalg/linalg.py#L21

Reply:
Import were corrected (at least, in files of this pull request)